src/Nav/Nav.js

```diff
@@ -1,27 +1,28 @@
 import React from 'react'
+import './Nav.scss'
 
 const Nav = (props) => {
   return (
-    <>
-    <button id='home-btn'>
-      Home
-    </button>
-    <button 
-      className={props.isLoggedIn ? '' : 'hidden'} 
-      id='ratings-header-btn'>
-      Your Ratings
-    </button> 
-    <button 
-      className={props.isLoggedIn ? 'hidden' : ''}
-      id='login-btn'>
-      Login
-    </button>
-    <button 
-      className={props.isLoggedIn ? '' : 'hidden'} 
-      id='logout-btn'>
-      Logout
-    </button>
-    </>
+    <nav>
```

Comment:
I still like the react fragments here - I think we can assign a class name to `<Nav />` when it's loaded in `App`